nva-datamodel-java/src/main/java/no/unit/nva/model/Publication.java

```diff
@@ -323,11 +324,28 @@ public String getJsonLdContext() {
 
     @JsonIgnore
     public boolean isPublishable() {
-        return !DRAFT_FOR_DELETION.equals(getStatus()) && nonNull(getEntityDescription()) &&
-                !isEmpty(getEntityDescription().getMainTitle()) && getAssociatedArtifacts().isPublishable();
+        return !DRAFT_FOR_DELETION.equals(getStatus()) && hasMainTitle() && hasReferencedContent();
 
     }
 
+    private boolean hasReferencedContent() {
+        return getAssociatedArtifacts().isPublishable() || hasOriginalDoi();
```

Comment:
Is OR correct here? Has DOI anything to do with referenced content?

Reply:
Implies the file is elsewhere